doc/tutorial.md

```diff
@@ -135,13 +148,18 @@ The additional parameters are:
 
 ![Script execution with adjusted __chequeLimit__](assets/polkadot.js_execute_script_buy.png)
 
-The flow of this code example is:
-* Bob: `initial_coin_minting`, but only once after publication.
-* Alice: `register_new_user`, but only once after publication.
-* Alice: `buy_coin`.
-* Alice: `wash_car` for each time you bought a washing coin.
 
-You have now successfully published a Move module on your Polkadot blockchain and executed successfully Move scripts.
+## Finishing the Tutorial
+
+You have successfully published a Move module on your Polkadot blockchain and executed a Move script successfully.
+
+You can iterate the full demonstration example for training purposes or personal interest. So far, the developer Bob has published his module for his car wash and initialized it by executing the script `initial_coin_minting`. It's time for Alice (Bob's first customer) to register, buy a coin and wash her car! Therefore, try to:
+- create transactions,
+- estimate the optimal amount of needed gas,
+- and execute them as user `Alice` for the following scripts:
+  - `register_new_user`,
+  - `buy_coin` (observe here Bob's and Alice's account balances, because the price of a washing coin is 1 UNIT),
+  - `wash_car`.
```

Comment:
```suggestion
You have successfully published a Move module on your Polkadot blockchain and executed a Move script successfully.

You can iterate the full demonstration example for training purposes or personal interest. So far, the developer Bob has published his module for his car wash and initialized it by executing the script `initial_coin_minting`. It's time for Alice (Bob's first customer) to register, buy a coin and wash her car! Therefore, try to:
- create transactions,
- estimate the optimal amount of needed gas,
- and execute them as user `Alice` for the following scripts:
  - `register_new_user`,
  - `buy_coin` (observe here Bob's and Alice's account balances, because the price of a washing coin is 1 UNIT),
  - `wash_car`.
```